.github/workflows/update-wiki.yml

```diff
@@ -0,0 +1,48 @@
+name: update wiki
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
```

Comment:
Does this mean that pull requests will already override our Wiki?

Reply:
I will remove it. I wanted to try running it from PR.